data/mods/classic_zombies/items/guns.json

```diff
@@ -88,5 +88,12 @@
     "name": { "str": "C6" },
     "copy-from": "m60",
     "description": "The Colt Canada C6 is a general-purpose machine gun developed to replace the .30-caliber C5 GPMG.  Heavy and difficult to handle fired from the shoulder, as most people aren't action-movie heroes."
+  },
+  {
+    "type": "GUN",
+    "id": "rossi_gallery",
```

Comment:
same ID as the rifle in vanilla? Shouldn’t this be a migration instead?

Reply:
This is essentially a migration, as it replaces it just with a new name. Look how other guns here do it. Prevents having to re-do item groups